jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -135,8 +141,11 @@ public JibContainerBuilder createJibContainerBuilder(
       JavaContainerBuilder javaContainerBuilder, ContainerizingMode containerizingMode) {
     try {
       if (isWarProject()) {
-        logger.info("WAR project identified, creating WAR image: " + project.getDisplayName());
-        Path explodedWarPath = GradleProjectProperties.getExplodedWarDirectory(project);
+        final String warFilePath = getWarFilePath();
+        logger.info("WAR project identified, creating WAR image from: " + warFilePath);
+        Path explodedWarPath = tempDirectoryProvider.newDirectory();
+        Path war = Paths.get(warFilePath);
+        ZipUtil.unzip(war, explodedWarPath);
```

Comment:
We can consolidate with less variables here:
```
ZipUtil.unzip(Paths.get(warFilePath), explodedWarPath);
```

Reply:
Done as suggested.